datasets/modeled/transport/dimension_mappings/scenario_to_scenario.csv

```diff
@@ -0,0 +1,3 @@
+from_id,to_id
+High+unmanaged_immediate,decarb_high
+High+unmanaged_smooth,
```

Comment:
Is this correct? If we aren’t going to use half the data, should we remove it for the dataset that gets registered?

Reply:
@elainethale says this is pending a further decision down the road and that we do want to do this 

              Yes, please do that for now.

_Originally posted by @elainethale in https://github.com/dsgrid/dsgrid-project-DECARB/pull/12#discussion_r1480650256_

Reply:
Arthur is right--we will use the data, we just haven't decided on an alternate scenario name yet.